builtin/providers/azurerm/config.go

```diff
@@ -0,0 +1,167 @@
+package azurerm
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure"
```

Comment:
It is correct - the Azure SDK vendors it's dependencies, but we need access to them for hooks.